### DISCUSSION TITLE
nbGrader and jupyterlab 4.2.2

### DISCUSSION THREAD

**Discussion by Mad31** [jupyter/nbgrader#1896] [General]
When i ask `jupyter labextension list` i have :
JupyterLab v4.2.2
/usr/local/share/jupyter/labextensions
        jupyterlab_pygments v0.3.0 enabled OK (python, jupyterlab_pygments)
        jupyterlab-myst v2.4.2 enabled OK (python, jupyterlab_myst)
        ipycanvas v0.13.2 enabled OK
        nbgrader v0.9.2 enabled  X (python, nbgrader)
        @jupyter-notebook/lab-extension v7.2.1 enabled OK
        @jupyter-widgets/jupyterlab-manager v5.0.11 enabled OK (python, jupyterlab_widgets)

   The following extensions may be outdated or specify dependencies that are incompatible with the current version of jupyterlab:
        nbgrader
        
   If you are a user, check if an update is available for these packages.
   See below for the details.

"nbgrader@0.9.2" is not compatible with the current JupyterLab
Conflicting Dependencies:
JupyterLab    Extension      Package
>=2.0.1 <3.0.0>=1.1.1 <2.0.0 @jupyter/ydoc

**But all seems to work well !**

**Comment by brichet** [comment]
Thanks @Mad31 for relating this.
It should be fixed on main, and released soon: https://github.com/jupyter/nbgrader/pull/1892
